### ISSUE TITLE
Note page showing an error

### ISSUE THREAD

**Issue by jywarren** [publiclab/plots2#2623]
https://publiclab.org/notes/karimatsko/3-17-2012/hydrogen-sulfide-h2s-oilgas

I'll check the logs -- no time at the moment! 👍

**Comment by icarito** [comment]
Sure!
```
Started GET "/notes/karimatsko/3-17-2012/hydrogen-sulfide-h2s-oilgas" for 190.238.17.50 at 2018-04-23 14:34:53 +0000
Processing by NotesController#show as HTML
  Parameters: {"author"=>"karimatsko", "date"=>"3-17-2012", "id"=>"hydrogen-sulfide-h2s-oilgas"}
Read fragment views/related_sidebar-feature (0.2ms)
  Rendered sidebar/_events.html.erb (4.1ms)
Started GET "/questions/tag/bh3" for 46.229.168.66 at 2018-04-23 14:34:54 +0000
Processing by TagController#show as */*
  Parameters: {"node_type"=>"questions", "id"=>"bh3"}
  Rendered users/profile.html.erb within layouts/application (850.2ms)
  Rendered layouts/_google_analytics.html.erb (0.0ms)
Read fragment views/feature_anniversary-banner (0.2ms)
Read fragment views/feature_sitewide-alert (0.1ms)
  Rendered layouts/_alerts.html.erb (0.4ms)
  Rendered layouts/_header.html.erb (3.4ms)
Read fragment views/feature_footer-notice (0.1ms)
  Rendered layouts/_footer.html.erb (0.3ms)
Completed 200 OK in 2083ms (Views: 494.9ms | ActiveRecord: 824.8ms)
  Rendered sidebar/_wikis.html.erb (126.1ms)
  Rendered sidebar/_notes.html.erb (67.3ms)
  Rendered sidebar/_related.html.erb (270.0ms)
  Rendered tag/_graph.html.erb (58.6ms)
  Rendered questions/_questions.html.erb (0.2ms)
  Rendered users/_answered.html.erb (0.0ms)
  Rendered tag/show.html.erb within layouts/application (61.6ms)
  Rendered layouts/_google_analytics.html.erb (0.0ms)
Read fragment views/feature_anniversary-banner (0.2ms)
Read fragment views/feature_sitewide-alert (0.1ms)
  Rendered layouts/_alerts.html.erb (0.3ms)
  Rendered layouts/_header.html.erb (3.1ms)
Read fragment views/feature_footer-notice (0.1ms)
  Rendered layouts/_footer.html.erb (0.2ms)
Completed 200 OK in 237ms (Views: 39.8ms | ActiveRecord: 100.2ms)
  Rendered like/_like.html.erb (53.7ms)
  Rendered notes/_gallery.html.erb (2.7ms)
  Rendered notes/show.html.erb within layouts/application (361.4ms)
Completed 500 Internal Server Error in 640ms (ActiveRecord: 372.8ms)

ActionView::Template::Error (undefined method `gsub' for nil:NilClass):
    66:   <%= render :partial => "notes/gallery" %>
    67:
    68:   <div id="content" class="pl-content note">
    69:   <%= raw auto_link(insert_extras(@node.latest.render_body), :sanitize => false) %>
    70:   </div>
    71:
    72:   <% if @node.files && @node.files.length > 0 %>
  app/models/concerns/node_shared.rb:32:in `notes_grid'
  app/helpers/application_helper.rb:45:in `insert_extras'
  app/views/notes/show.html.erb:69:in `_app_views_notes_show_html_erb__451220027155206227_60134580'


```

**Comment by jywarren** [comment]
I think this'll be fixed by #2651 -- closing this!

**Comment by jywarren** [comment]
And thanks!
